Assets/Scripts/Modules/Combat/ICombatModule.cs

```diff
@@ -1,8 +1,19 @@
 ﻿/*
  * Authors: Ian Macpherson, Isaiah Mann
- * Description: An interface to handle the turn logic
+ * Description: An interface to handle the combat logic
  */
 
 public interface ICombatModule {
 
+	bool IsTargetInRange (IUnit unit);
```

Comment:
This module should be a single standalone, not per unit. Therefore is should take in both a target and an attacker.

I could definitely see creating another interface called something along the lines of "IUnitCombatModule" that would contain methods like this one